### PR TITLE
Farsight DNSDB Open Source Submission Pull Request

### PR DESCRIPTION
**Pull request by RamboV**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by RamboV** [comment]
@sriyen-msft 

You can start the review.

Thanks.

File: certified-connectors/Farsight Security/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,898 @@
...
+              },
+              "X-RateLimit-Expires": {
+                "description": "when period expires",
+                "type": "number"
+              }
+            }
+          }
+        },
+        "summary": "RRSet Lookup",
+        "description": "The \u201crrset\u201d lookup queries DNSDB\u2019s RRset index, which supports \u201cforward\u201d lookups based on the owner name of an RRset.",
```
[the remark this answers is not in the document]
Comment (sriyen-msft) [reply certified-connectors/Farsight Security/apiDefinition.swagger.json]: @RamboV I can still see unicodes in the swagger, Please find them in the editor with `\u`

[the remark this answers is not in the document]
Comment (RamboV) [reply certified-connectors/Farsight Security/apiDefinition.swagger.json]: @sriyen-msft 
Thanks, I guess all should be fixed now, please review. 
Guess the paconn plugin is converting few characters to Unicodes.

File: certified-connectors/Farsight Security/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Farsight DNSDB Connector
+
+Farsight Security DNSDB® is the world’s largest DNS intelligence database that provides a fact-based view of the configuration of the global Internet infrastructure. DNSDB leverages Farsight’s Security Information Exchange (SIE) data-sharing platform and is engineered and operated by leading DNS experts. Farsight collects, filters, and verifies Passive DNS data from its global sensor array. DNSDB is the highest-quality and most comprehensive DNS intelligence data service of its kind.
```
Comment (sriyen-msft) [inline certified-connectors/Farsight Security/README.md:3]: @RamboV Adding copy rights and register symbols in any connector file (including readme.md and intro.md) is not allowed, Our legal team will not let us certify with either the copyright or registered trademarks anywhere in the connector artifacts or files as we don't want to imply trademark or copyright. Please remove all the copyright content from all the connector files.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Farsight Security/README.md (hunk 2 above)
Comment (RamboV) [reply certified-connectors/Farsight Security/README.md]: @sriyen-msft Thanks for the Information. I have modified the REAdME.md accordigly

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**